tests/integration/graviton_test.py

```diff
@@ -510,9 +494,11 @@ def blackbox_test_runner(
     algod = algod_with_assertion()
 
     # 2. validate dry run scenarios:
-    inputs, predicates = Invariant.inputs_and_invariants(
```

Comment:
This error-prone method is no longer available

Reply:
👍